frontend/public/components/monitoring/alerting.tsx

```diff
@@ -24,8 +28,10 @@ import {
   OutlinedBellIcon,
 } from '@patternfly/react-icons';
 
-import { useActivePerspective, ResourceStatus } from '@console/dynamic-plugin-sdk';
+import { useActivePerspective, ResourceStatus, Action } from '@console/dynamic-plugin-sdk';
 import {
+  ActionContext,
```

Comment:
I think this can be imported from `dynamic-plugin-sdk` instead, which would better align with the changes in https://github.com/openshift/console/pull/11784.

Reply:
@kyoto the type can be easily imported from the SDK but `ActionServiceProvider`, which loads the extension point as a hook, is still inside `@console/shared`.  Maybe it would make sense to move/reimplement it in the SDK as it will allow other plugins to define `console.action/provider` extension points. WDYT @spadgett?

Reply:
I imagine it makes sense to expose `ActionServiceProvider` from the SDK. It's quite widely used in the Console codebase, so likely a useful thing to expose.

CC @spadgett

Reply:
Right, it looks like we'll need it for other plugins as well searching the code. @christianvogt @vojtechszocs thoughts?

Reply:
probably makes sense. Removes boilerplate for anyone wanting to consume action extensions